composer.json

```diff
@@ -17,7 +17,7 @@
 		}
 	},
 	"require": {
-		"php": ">=7.1",
+		"php": ">=7.4",
```

Comment:
Well, this is a BC break. Could you release a mojor version? Or revert this change.